context/app/static/js/components/Detail/FileBrowser/FileBrowser.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import FileBrowserNode from '../FileBrowserNode';
+import { ScrollPaper } from './style';
+
+function FileBrowser(props) {
+  const { files } = props;
+  const [fileTree, setFileTree] = useState({});
+  useEffect(() => {
+    const treePath = {};
+    files.forEach((f) => {
+      const levels = f.rel_path.split('/');
+      const file = levels.pop();
+
+      const fileObj = { file, fullPath: f.rel_path, description: f.description, edam_term: f.edam_term, size: f.size };
```

Comment:
Spread syntax?

Reply:
Or, I guess, be consistent about camel-case

Reply:
Agreed, I'll fix it.